index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="public/stylesheets/styles.css">
+
+  <title>Calculator Benchmark</title>
+
+</head>
+<body>
+  <div class="calculator-container">
+    <div>
+      <div class="calculator-body">
+        <div class="calculator-window">
+            <div class="btn-window btn-close btn-spacing">x</div>
+            <div class="btn-window btn-min btn-spacing">-</div>
+            <div class="btn-window btn-open btn-spacing">+</div>
+        </div>
+        <div class="calculator-nums">
+          <div class="calculator-row">
+            <div class="calculator-button calculator-num-colors">
```

Comment:
all the divs with class `. calculator-button` should probably be `<button>` elements